WORKSPACE

```diff
@@ -569,14 +594,14 @@ hazel_repositories(
                 "bubba",
                 "lsp-test",
                 "d126623dc6895d325e3d204d74e2a22d4f515587",
-                "a2be2d812010eaadd4885fb0228370a5467627bbb6bd43177fd1f6e5a7eb05f8",
+                "214848612d319bbded67341c51fd151d65cc80264b9cfe70755ef6941b450ec9",
```

Comment:
The reason why all of the hashes changed is that I switched `hazel_github_external` from `.zip` to `.tar.gz` since Bazel falls over on the symlink in gRPC-haskell otherwise (https://github.com/bazelbuild/bazel/issues/2656 has the same error but is closed for some reason).